app/controllers/rentals_controller.rb

```diff
@@ -30,6 +30,17 @@ def confirm
     end
   end
 
+  def rate_salon
+    render 'rentals/rate_salon'
+  end
+
+  def update_rating
+    @rental = Rental.find(params[:rental_id])
+    @rental.update(rental_params)
```

Comment:
Here you can update any field of rental since you permitted all params. You need to have something like rental_rating_params method with only ratings permitted.